core/templates/components/code-mirror/codemirror-mergeview.component.ts

```diff
@@ -64,8 +66,10 @@ export class CodemirrorMergeviewComponent implements
         throw new Error('Left pane value is not defined.');
       }
       this.ngZone.runOutsideAngular(() => {
-        this.codeMirrorInstance.editor().setValue(
-          changes.leftValue.currentValue);
+        if (this.codeMirrorInstance !== undefined) {
```

Comment:
We should also cater what happens if its value is `undefined`

Reply:
done